packages/qwik/src/core/render/dom/render.unit.tsx

```diff
@@ -1113,3 +1126,15 @@ test('should render value="" on option', async () => {
   assert.equal(option.getAttribute('value'), '');
   assert.equal(option.outerHTML, '<option value="">Empty</option>');
 });
+
+async function throws<T>(fn: () => T, expected?: string): Promise<void> {
```

Comment:
vitest's expect() already has https://vitest.dev/api/expect.html#tothrowerror, use that instead